tests/AttributeBagTest.php

```
@@ -0,0 +1,149 @@
+     */
+    #[DataProviderExternal(AttributeBagProvider::class, 'set')]
+    public function testSet(
+        array $attributes,
+        string|UnitEnum $key,
+        bool|float|int|string|Closure|Stringable|UnitEnum|null $value,
+        string $prefix,
+        bool $boolToString,
+        array $expected,
+    ): void {
+        match ($boolToString) {
+            true => AttributeBag::set($attributes, $key, $value, $prefix, true),
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Helper\AttributeBag' in method 'testSet'. (undefined)

(StaticAccess)